src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1615,7 +1641,7 @@ private void doNetworkWindow(
             state = AmqpState.openInitial(state);
 
             initialBudget += credit;
-            doWindow(network, routeId, initialId, traceId, authorization, budgetId, credit, padding);
+            doWindow(network, routeId, initialId, traceId, authorization, budgetId, credit, padding, 0);
```

Comment:
Define a method called `doNetworkWindow` to make this part of the code more readable.

Reply:
This `doWindow(..)` call is made inside `doNetworkWindow` method. Are you suggesting to create another `doNetworkWindow` method that takes in different parameters?

Reply:
sorry, that context was lost on me in the diff, it's all good - already done 👍